ignite/contrib/handlers/param_scheduler.py

```diff
@@ -553,6 +553,47 @@ def create_lr_scheduler_with_warmup(lr_scheduler, warmup_start_value, warmup_end
     return combined_scheduler
 
 
+class ParamGroupScheduler(object):
+    """
+    Scheduler helper to group multiple schedulers into one.
+
+    Args:
+        schedulers (list/tuple of ParamScheduler): list/tuple of parameter schedulers.
+        names (list of str): list of names of schedulers.
+
+    .. code-block:: python
+
+        names = []
+        lr_schedulers = []
+        for i, param_group in enumerate(optimizer.param_groups):
+            names.append("param_group_{}".format(i))
+            lr_schedulers.append(get_lr_scheduler(param_group, param_group_scheduling_conf[i]))
```

Comment:
Maybe add a comment giving an example of get_lr_scheduler? or just an example of what this line might look like. This can give users a starting point.

Reply:
@anmolsjoshi improved the example. Take a look, plz